eis_toolkit/cli.py

```diff
@@ -605,6 +605,31 @@ def unify_rasters_cli(
     typer.echo(f"Unifying completed, writing rasters to {output_directory}.")
 
 
+# GET UNIQUE COMBINATIONS
+@app.command()
+def unique_combinations_cli(
+    input_rasters: Annotated[List[Path], INPUT_FILE_OPTION],
+    output_directory: Annotated[Path, OUTPUT_DIR_OPTION],
```

Comment:
Change into `output_raster`. This will be the full path that you can directly put to the writing command. So, follow the same formula as in other raster processing CLI functions.